frontend/src/lib/components/PropertyIcon.tsx

```diff
@@ -51,6 +51,7 @@ export const PROPERTIES_ICON_MAP = {
         ['desktop']: <IconMonitor />,
         ['mobile']: <IconPhone />,
         ['tablet']: <IconTablet />,
+        ['console']: <IconHeadset />,
         ['other']: <IconDevices />,
```

Comment:
I couldn't find any good icon for wearables either, I'm happy with `IconDevices`, unless @corywatilo is keen to draw us one - @lricoy you could use the art/icon request on the posthog.com repo for this

Reply:
Sounds good, I will probably batch some other requests for channel types as well.

Reply:
I have this one out already, would love if that could be part of the batch as well: https://github.com/PostHog/posthog.com/issues/10542